acrg/name/name.py

```diff
@@ -288,38 +302,31 @@ def footprints(sitecode_or_filename, met_model = None, fp_directory = None,
     # into an annual footprint file in (mol/mol) / (mol/m2/s)
     # using acrg_name_process
     
-    if fp_directory is None:
-        fp_integrated_directory = join(data_path, 'LPDM/fp_NAME/')
-        fp_HiTRes_directory = join(data_path,'LPDM/fp_NAME_high_time_res/')
-        fp_directory = {'integrated': fp_integrated_directory,
-                        'HiTRes': fp_HiTRes_directory}
-
-    #   Error message if looking for HiTRes files and fp_directory is not a dictionary        
-        if HiTRes is True:
-            if type(fp_directory) is not dict:
-                print("fp_directory needs to be a dictionary containing paths \
-                       to integrated and HiTRes footprints \
-                       {integrated:path1, HiTRes:path2}")
-                return None
+    fp_directory = join(data_path, 'LPDM/fp_NAME/') if fp_directory is None else fp_directory
```

Comment:
Based on previous code - is it no longer expected that `fp_directory` will be a dictionary for 'HiTRes' option?

Reply:
I've never used a dictionary for it as the HiTRes footprints are stored in the same area now, do you think it would be better to change the code to make that an option again, or just take out reference to is being a dictionary?

Reply:
A lot of the code would need changing to use a dictionary for it, and I don't think you'd often have a reason for importing both so I think it would be best just to change the documentation
(The way the footprints are processed now means that the HiTRes code now only needs the HiTRes ones)

Reply:
I think the question is around backwards compatibility - thinking about it I think the dictionary option was because we used to have to use separate integrated and HiTRes files. If it's not too much work if may be good to allow it to be included as an option? Otherwise, maybe you could just add a comment to say this mode is no longer supported.

Reply:
Yeah ok I think that's fine - seems like a thing to add to the [change log](https://github.com/ACRG-Bristol/acrg/blob/develop/CHANGELOG.md) just so we know when the change happened.

Reply:
Sounds good